source/agora/test/EnrollmentManager.d

```diff
@@ -89,21 +89,8 @@ unittest
     txs2.each!(tx => node_1.putTransaction(tx));
     containSameBlocks(nodes, 2).retryFor(8.seconds);
 
-    // Currently a new transaction needs to be sent to trigger
-    // the reveal of a pre-image
-    // See https://github.com/bpfkorea/agora/issues/582
-    Transaction tx =
-    {
-        TxType.Payment,
-        [Input(hashFull(txs2[0]), 0)],
-        [Output(Amount(100), pubkey_1)]
-    };
-    auto signature = gen_key_pair.secret.sign(hashFull(tx)[]);
-    tx.inputs[0].signature = signature;
-    node_1.putTransaction(tx);
-
-    // check if nodes have a pre-image newly sent
-    // during creating transactions for the new block
```

Comment:
Isn't it great to remove workaround code ? :D

However the `retryFor` is for 5 seconds only, isn't there a risk that the image isn't propagated ?